v2/brigadier/src/jobs.ts

```diff
@@ -20,33 +61,177 @@ export class Job {
+   * are treated as arguments. For example, a command of ["echo", "hello"] is
+   * equivalent to running 'echo hello'. A common convention is to use the command
+   * array for subcommands and the arguments array for argument values.
+   * 
+   * @example
+   * job.command = ["helm", "install"]
+   * job.arguments = ["stable/nginx", "-g"]
+   */
   public command: string[] = []
+  /**
+   * The arguments to pass to Container#command.  If the command includes arguments
+   * already, the arguments property are appended. A common convention is to use the command
+   * array for subcommands and the arguments array for argument values.
+   * 
+   * @example
+   * job.command = ["helm", "install"]
+   * job.arguments = ["stable/nginx", "-g"]
+   * 
+   */
   public arguments: string[] = []
+  /** Environment variables to set in the container. */
   public environment: { [key: string]: string } = {}
+  /**
+   * The path in the container's file system where, if applicable, the
+   * Brigade worker's shared workspace should be mounted. If empty (the default),
+   * the Job does not have access to the shared workspace.
+   */
   public workspaceMountPath = ""
+  /**
+   * The path in the container's file system where, if applicable,
+   * source code retrieved from a version control system repository should be
+	 * mounted. If empty (the default), Brigade will not mount any source
+   * code automatically.
+   */
   public sourceMountPath = ""
+  /**
+   * Whether the container should run with privileged permissions. This is
+   * typically required only for "Docker in Docker" scenarios where the
+   * container must run its own Docker daemon.
+   * 
+   * Privileged execution may be disallowed by Brigade project configuration.
+   * If so, the container will run unprivileged.
+   */
   public privileged = false
+  /**
+   * Whether the container should mount the host's Docker socket into its own
+   * file system. This is typically required only for "Docker-out-of-Docker" ("DooD")
+   * scenarios where the container needs to use the host's Docker daemon.
+   * This is strongly discouraged for almost all use cases.
+   * 
+   * Host Docker socket access may be disallowed by Brigade project configuration.
+   * If so, the container will run without such access.
+   */
   public useHostDockerSocket = false
 
+  /**
+   * Constructs a new Container.
+   * @param image The OCI reference to the container image
+   */
   constructor(image: string) {
     this.image = image
   }
 }
 
+/**
+ * The execution environment required by a Job.
+ */
 export class JobHost {
+  /**
+   * The OS required by the Job. Valid values are "linux" and "windows". When empty,
+	 * Brigade assumes "linux".
+   */
   public os?: string
+  /**
+   * Specifies labels that must be present on the substrate node to
+	 * host a Job. This provides an opaque mechanism for communicating Job needs
+	 * such as specific hardware like an SSD or GPU.
```

Comment:
Indent is off.

Reply:
Think I had some rogue tabs from somewhere (maybe I copy-pasted from the Go docs!).  Hopefully all gone now.